NuRadioReco/examples/RNO_data/read_data_example/run_reconstruction.py

```diff
@@ -0,0 +1,71 @@
+import NuRadioReco
+import matplotlib.pyplot as plt
+from NuRadioReco.modules.io.rno_g.readRNOGData import readRNOGData
+import pandas as pd
+import numpy as np
+from NuRadioReco.utilities import units
+from NuRadioReco.modules import channelBandPassFilter
+from NuRadioReco.detector import detector
+import datetime
+from NuRadioReco.modules import sphericalWaveFitter
+from NuRadioReco.modules import channelAddCableDelay
+
+""" An example to show how to read RNO-G data and how to perform simple reconstructions. The data used is pulser data and a simple (brute force, not optimized) spherical wave reconstruction is performed to obtain the pulser position """""
+
+""" Initiazize modules needed"""
+
+channelBandPassFilter = NuRadioReco.modules.channelBandPassFilter.channelBandPassFilter()
+sphericalWaveFitter = NuRadioReco.modules.sphericalWaveFitter.sphericalWaveFitter()
+channelAddCableDelay = NuRadioReco.modules.channelAddCableDelay.channelAddCableDelay()
+
+
+use_channels = [10,9, 0,3, 21, 22]
```

Comment:
I don't know which one you want to keep, but except for the `use_channels` line `NuRadioReco/examples/RNO_data/read_data_example/run_reconstruction.py` and `NuRadioReco/examples/RNO_data/run_reconstruction.py` are identical

(if you keep this one, maybe you want to add some spaces for beauty `use_channels = [10, 9, 0, 3, 21, 22]`